assets/mods/simplify/postloadModule.js

```diff
@@ -2,7 +2,10 @@ import * as patchSteps from './lib/patch-steps-lib/src/patchsteps.js';
 import CustomDebugState from './lib/custom-debug-state.js';
 (() => {
 	const igroot = window.IG_ROOT || '';
-	
+
+	// Should this be added to preload? 
+	window.registerCallable = patchSteps.registerCallable;
```

Comment:
The name `registerCallable` on it's own does not give enough information to figure out what it does.